leaderelection/context.go

```diff
@@ -92,6 +98,9 @@ func BuildElector(ctx context.Context, la reconciler.LeaderAware, queueName stri
 		}
 	}
 
+	universe := reconciler.UniversalBucket()
+	la.Promote(universe, nil)
```

Comment:
I think you only meant to do this or the other stuff

Reply:
oh whoops - thanks